### ISSUE TITLE
route not adding base path for OpenAPIHono

### ISSUE THREAD

**Issue by Amorim33** [honojs/middleware#290]
## Description

After updating `hono` to the version `3.11.2`, `route` method stopped adding the base path

```ts
const app = new OpenAPIHono<{
    Bindings?: AppBindings & TEnv['Bindings'];
    Variables: TEnv['Variables'];
  }>();

app.route('/sub-path', createMyRouter());
```
> createMyRouter returns an OpenAPIHono instance

```ts
showRoutes(app)
```
![image](https://github.com/honojs/middleware/assets/42624869/b59a51ee-4b6a-46fb-86cf-6949be985f1d)

## Context
```json
"hono": "^3.11.2",
"@hono/zod-openapi": "^0.9.2"
```

**Comment by yusukebe** [comment]
Hi @Amorim33 

Thanks for raising the issue. This might be fixed with the latest version of `hono@3.11.3`. Please update the `hono` and try it.

**Comment by Amorim33** [comment]
Thank you, very much!🙏
